_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -48,7 +54,101 @@ library(tidyverse)
+We can see that two of the columns are characters (categorical variables), and three are doubles (numbers).
+
+**Note** In anything before R 4.0, categorical variables used to be read in as factors, which are a [special data object](https://www.tutorialspoint.com/r/r_factors.htm) that are used to store categorical data and have limited numbers of unique values. The unique values of a factor are tracked via the "levels" of a factor. A factor will always remember all of its levels even if the values don't actually appear in your data. The factor will also remember the order of the levels and will always print values out in the same order (by default this order is alphabetical).
+
+If your columns are stored as character values but you need factors for plotting, ggplot will convert them to factors for you as needed.
+
+
+Now let's look at the data a bit more. 
+In the **Environment** tab in the upper right corner of RStudio, lou will now see `gapminder_1997` listed.
+If you click on it, it will pop up in a tab next to your script. 
+
+After we've reviewed the data, you'll want to make sure to click the tab in the upper left to return to your `gdp_population.R` file so we can start writing some code.
+
+Another way to look at the data is to print it out to the console:
+
+```{r, viewGapminder}
+gapminder_1997
+```
+
+The `read_csv()` function took the file path we provided, did who-knows-what behind the scenes, and then outputted an R object with the data stored in that csv file. All that, with one short line of code!
+
+> ## Data objects
+> There are many different ways to store data in R. Most objects have a table-like structure with rows and columns. We will refer to these objects generally as "data objects". If you've used R before, you many be used to calling them "data.frames". Functions from the "tidyverse" such as `read_csv()` work with objects called "tibbles", which are a specialized kind of "data.frame." Another common way to store data is a "data.table". All of these types of data objects (tibbles, data.frames, and data.tables) can be used with the commands we will learn in this lesson to make plots. We may sometimes use these terms interchangeably.
+{: .callout}
+
+> ## Bonus: Reading in an excel file
+> Say you have an excel file and not a csv - how would you read that in? Hint: Use the Internet to help you figure it out!
+>
+> {: .source}
+>
+> > ## Solution
+> > One way is using the `read_excel` function in the `readxl` package. There are other ways, but this is our preferred method because the output will be the same as the output of `read_csv`.
+> {: .solution}
+{: .challenge}
+
+
+# Our first plot
+_[Back to top](#contents)_
+
+## Creating our first plot
 _[Back to top](#contents)_
 
 We will be using the `ggplot2` package today to make our plots. This is a very
```

Comment:
As noted above, I think we need to be careful in how we introduce aesthetics (lines 178 - 181). I think some more clear language would be:

The elements of a plot have a bunch of properties like an x and y position, a size, a color, etc. When creating a data visualization, we can map variables in our dataset to these properties, called **aesthetics**, in our plot. In ggplot, we can do this by creating an "aesthetic mapping", which we do with the `aes()` function.

Might want to bold "geometry" on line 253.